shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -109,7 +109,7 @@ - (instancetype)init {
 - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridge>)bridge uid:(int32_t)uid {
   FML_DCHECK(bridge) << "bridge must be set";
   FML_DCHECK(uid >= kRootNodeId);
-  self = [super init];
+  self = [super initWithAccessibilityContainer:bridge->view()];
```

Comment:
Here and everywhere else this is called: I believe not all SemanticsObjects have the view as an accessibility container. There are some intermediate a11y containers that we create. Shouldn't the SemanticsObjects be attached to their container, not the root?

Reply:
The problem is we don't know the containers at this point - but it seems to work because we later assign it to the right container when we actually create the container.

Reply:
There should probably at least be a comment here then. And I am wondering how we can assure that it is going to be assigned the right container....